app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Manage Dev Portal Teams
+content_type: how-to
+---
+
+A common scenario for a Dev Portal is to restrict developers' ability to view or request access to specific services based on their permissions. {{site.konnect_short_name}} Portal enables administrators to define Role-Based Access Control (RBAC) for teams of developers through the Konnect UI and API.
+
+Portal RBAC supports two different roles for services that can be applied to a team of developers:
+
+* **API Viewer:** Provides developers with read access to the documentation associated with a service.
+* **API Consumer:** Includes API Viewer permissions as well as allows developers to register applications to consume versions of a service.
+
+You can use the API and UI to create a team, assign a role to the team, and finally, add developers to the team.
+
+In this guide, you will set up two developer teams and then enable Portal RBAC using a hypothetical scenario.
+
+## Prerequisites
+* Two [test developer accounts registered](/konnect/dev-portal/dev-reg/)
+* A [runtime configured](/konnect/getting-started/configure-runtime)
+
+## Configure developer teams 
+
+In this scenario, you are a product manager at a pizza company who is in charge of their online app. You're asked to create a Dev Portal that allows delivery companies to use your APIs to deliver the company's pizzas. However, you want to make sure only trusted delivery partners can build apps using your APIs and deliver your pizzas. 
```

Comment:
❗ For reviewers: Does this scenario make sense? Is there anything I should change?

Reply:
It does with some tweaks I made as suggestions